lib/cheetah.rb

```diff
@@ -308,6 +309,8 @@ class << self
     #   @option options [Hash] :env ({})
     #     Allows to update ENV for the time of running the command. if key maps to nil value it
     #     is deleted from ENV.
+    #   @option options [String] :chroot ("/")
+    #     Allows to run on different system root.
```

Comment:
exception is raised and due to improvement in catching system error it is written to pipe, so user see it. For my testing script:

```
jreidinger@pepa:~/cheetah> ruby test.rb 
/home/jreidinger/cheetah/lib/cheetah.rb:641:in `check_errors': Execution of "touch /aaa" failed with status 127: Operation not permitted @ dir_s_chroot - /tmp/root. (Cheetah::ExecutionFailed)
	from /home/jreidinger/cheetah/lib/cheetah.rb:404:in `run'
	from test.rb:3:in `<main>'
```